packages/near-membrane-base/src/membrane.ts

```diff
@@ -109,6 +109,11 @@ export type HooksCallback = (
     callableGetTargetIntegrityTraits: CallableGetTargetIntegrityTraits,
     callableHasOwnProperty: CallableHasOwnProperty
 ) => void;
+// eslint-disable-next-line no-shadow
+export enum SupportFlags {
+    None = 0,
+    MagicMarker = 1 << 0,
+}
```

Comment:
I love this

Reply:
it is a shame that we have to duplicate this inside the init as well :(

Reply:
> it is a shame that we have to duplicate this inside the init as well :(

Such is life in the serialized function. Can't use object spread in it either.